simphony/core/cuba.yml

```diff
@@ -480,4 +480,46 @@
   name: Material
   number: 68
   shape: [1]
-  type: uuid
\ No newline at end of file
+  type: uuid
+- description: Cutoff Distance
+  domain: []
+  key: CUTOFF_DISTANCE
+  name: CutoffDistance
+  number: 69
+  shape: [1]
+  type: double
+- description: Energy Well Depth
+  domain: []
+  key: ENERGY_WELL_DEPTH
+  name: EnergyWellDepth
+  number: 70
+  shape: [1]
+  type: double
+- description: Van Der Waals Radius
+  domain: []
+  key: VAN_DER_WAALS_RADIUS
+  name: VanDerWaalsRadius
+  number: 71
+  shape: [1]
+  type: double
+- description: Dielectric Contance
+  domain: []
+  key: DIELECTRIC_CONTANCE
+  name: DielectricContance
+  number: 72
+  shape: [1]
+  type: double
+- description: Lennard Jones
```

Comment:
I think down the road maybe we could have a CUBA list which contained everything (including material relations). But currently we use the CUBA list as the list of everything that can go in a 'DataContainer'.   So until something like #29 happens, I think it would be best to have a separate enum for just the MaterialRelations (i.e. COULOMB and LENNARD_JONES)